utils/clients/client.go

```diff
@@ -216,7 +218,7 @@ func (c *SimpleHTTPClient) do(
 	// // helpful if you want to read the body as it is
 	// bodyBytes, _ := requestutils.Read(resp.Body)
 	// resp.Body.Close() // must close
-	// fmt.Println(string(bodyBytes))
+	// fmt.Println(req.URL.Host, req.URL.Path, string(bodyBytes))
 	// resp.Body = ioutil.NopCloser(bytes.NewBuffer(bodyBytes))
```

Comment:
Comment needed?

Reply:
useful to keep just because i often need to see what is going wrong under the hood.